### ISSUE TITLE
Cosmetic Issue > Dashboard > Upcoming Emails > Displayed scheduled date/time is wrong

### ISSUE THREAD

**Issue by quadcom** [mautic/mautic#10888]
### Mautic Version

4.2.x series

### PHP version

7.4

### What browsers are you seeing the problem on?

Chrome

### What happened?

I noticed on the dashboard "Upcoming Emails" widget that there was an email scheduled to send. 

![image](https://user-images.githubusercontent.com/7085931/155345377-db4eee5b-7247-4d9e-a566-66a97b67f52a.png)

I checked my program that had a step to only send emails between 9:00 pm and 10:00 pm. That was still set correctly. 

![image](https://user-images.githubusercontent.com/7085931/155345458-5f4e98bc-150f-4ab5-a6f8-fd2320861413.png)

I then checked the contact record and can see the scheduled email in the Events List with the correct date/time.
![image](https://user-images.githubusercontent.com/7085931/155345643-1a586577-d689-4c1b-93e6-c7213a3946fb.png)

### How can we reproduce this issue?

schedule and email to be sent and check the dashboard to see if the scheduled date/time is different.

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

**Comment by quadcom** [comment]
Another one came in and it is showing the exact same delivery date/time

![image](https://user-images.githubusercontent.com/7085931/155442345-34355fca-f179-40f7-9216-b81bdecc1944.png)

But on the contact record, it is showing the right send date/time

![image](https://user-images.githubusercontent.com/7085931/155442462-17134652-792d-4496-9182-4c5dae869d25.png)

**Comment by RCheesley** [comment]
Thanks for the bug report @quadcom - so from my understanding the dashboard widget is not taking into account the rules which are set for the times that emails may be sent in campaigns?  They do actually get sent at the right time, but the widget is showing the schedule date incorrectly?

**Comment by quadcom** [comment]
I can't be sure as I don't know how the system underneath works. Both contacts were created that same day but not at the same time so it's not showing a contact creation date/time. The campaign is set to send out emails between 9 pm and 10 pm on the day the contact was added to the campaign. Being these two upcoming events have the exact same time in the widget, it leads me to suspect a timezone offset is either is or is not being applied.

The time difference between when the email are going out and the time shown in the widget is 5 hours. I'm in EST which is GMT -5. I've checked my OS timezone and that's set right. It appears GMT might be being used in some way here.
